utils/sl/check_script.py

```diff
@@ -220,44 +209,33 @@ def check_script(fname, *args, **kw):
         pylint_path = "{path}"
 
     # Python 2 checking is no longer supported
-    pycmd = find_executable("pylint-3")
-    rc_file = "tmp_pylint3.rc"
+    pycmd = find_executable('pylint')
     if pycmd is None:
-        print("Required pylint isn't installed on this machine")
-        return
+        pycmd = find_executable('pylint-3')
+        if pycmd is None:
+            print("Required pylint isn't installed on this machine")
+            return
 
+    rc_file = "tmp_pylint3.rc"
     rc_dir = os.path.dirname(os.path.realpath(__file__))
 
-    cmd = pycmd.split() + \
-          list(args) + \
-          ["--rcfile=%s/%s" % (rc_dir, rc_file),
-           "--unsafe-load-any-extension=y",
-           "--msg-template",
-           "{C}: %s:{line}: pylint-{symbol}: {msg}" % pylint_path,
-           tmp_fname]
-
-    if os.environ.get("DEBUG_CHECK_SCRIPT", 0):
-        print(" ".join(cmd))
+    cmd = pycmd.split() + list(args) + \
+        ["--rcfile=%s/%s" % (rc_dir, rc_file),
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string